### DISCUSSION TITLE
About the installation of files

### DISCUSSION THREAD

**Discussion by justforlxz** [Homebrew/discussions#2535] [Writing Formulae/Casks] [answered]
Hi, everyone, I am the developer of deepin. We have a style plug-in developed based on qt, which can work on mac, but I hope to be added to brew so that more people can install it easily.

The question now is that the style plug-in needs to be placed in the plugins directory of qt. Can i install the file in the directory of other software packages?

Thanks！

**Comment by SMillerDev** [comment, marked as answer]
You can't write to the install directory of another formula, but you can write to common locations like `/usr/local/share`
